src/ClientWidgetApi.ts

```diff
@@ -372,8 +372,6 @@ export class ClientWidgetApi extends EventEmitter {
             }
         }
 
-        const limit = request.data.limit || 0;
```

Comment:
I'm not clear on why this is being removed? The WidgetDriver docs say that a `0` limit implies the widget is requesting as many events as possible, so it should be supplied with a number rather than undefined.